app/mixins/editor-base-controller.js

```diff
@@ -331,7 +331,16 @@ export default Mixin.create({
 
         notifications.showAlert(message, {type: 'error', delayed: delay, key: 'post.save'});
     },
-
+    keyDown: task(function* (event) {
+        switch (event.keyCode) {
+            // if the enter key or down key are pressed then focus on the editor
+            case 13:
+            case 40:
+                Ember.$('.__mobiledoc-editor').focus();
```

Comment:
If you want access to jQuery you can use `import $ from 'jQuery'` if that will help with the linting problems.

The `editor-base-controller` mixin is unfortunately a bit of a mess, in many older areas of our codebase we have controllers taking on the responsibilities of a component.

I may have missed it but what is the reasoning behind using an ember-concurrency task here? As far as I can see there's no asynchrony involved - would it make more sense for this to be a standard action?